<launch>

	<arg name="p_gain" value="150"/>
	<arg name="i_gain" value="0"/>
	<arg name="d_gain" value="0"/>

	<arg name="j0" value="180.0"/>
	<arg name="j1" value="180.0"/>
	<arg name="j2" value="180.0"/>
	<arg name="j3" value="180.0"/>
	<arg name="j4" value="180.0"/>
	<arg name="j5" value="180.0"/>
	<arg name="j6" value="180.0"/>

	<node name="pid_jaco" pkg="iact_control" type="pid_jaco.py" args="$(arg p_gain) $(arg i_gain) $(arg d_gain) $(arg j0) $(arg j1) $(arg j2) $(arg j3) $(arg j4) $(arg j5) $(arg j6)" output="screen"/>

</launch>
